Mathlib/Tactic/ProxyType.lean

```diff
@@ -229,7 +229,7 @@ def ensureProxyEquiv (config : ProxyEquivConfig) (indVal : InductiveVal) : TermE
           levelParams := indVal.levelParams
           safety := DefinitionSafety.safe
           hints := ReducibilityHints.abbrev
-          type := (← inferType equiv')
+          type := ← inferType equiv'
```

Comment:
Could @eric-wieser confirm that this is still correct?

Reply:
Yes, there is no other possible interpretation here other than a syntax error, which CI would catch.